packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -281,16 +281,26 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
     );
 
     // If the wallet queues a message, send it across the wire
-    this.paymentChannelClient.onMessageQueued((message: Message) => {
+    this.paymentChannelClient.onMessageQueued((message: Message | any) => {
+      // TODO remove any
```

Comment:
Why could it be `any`?

Reply:
Good question, this is what I am trying to figure out. I put the `any` here otherwise my runtime check is a type error: it's a reflection that something other than a `Message` is sometimes getting passed to this function.